main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -1,56 +1,91 @@
+chain-specific methods. See [getChain](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#getChain).
 
-A key advantage of ICAs is that they make accounts on other chains look like any other (remotable) object. When a contract creates an ICA, it has sole access to and control over the account but can delegate certain forms of access to its clients.
+```javascript
+const chain = await orchestrator.getChain('chainName');
+```
 
-For a detailed explanation of these access control rules, see [Access Control with Objects](/guides/zoe/contract-access-control).
+### Create an Account 
+- `makeLocalAccount` creates a new account on local chain. See [makeLocalAccount](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#makeLocalAccount).
 
-### Example ICA Usage from a Smart Contract
+```javascript
+const localAccount = await orchestrator.makeLocalAccount();
+```
 
-This sample is taken from one of the [example contracts](https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/examples/swapExample.contract.js)
+### Brand Utility Functions
+- `getBrandInfo` returns information about a `denom`, including the equivalent local Brand, 
+the chain where the denom is held, and the chain that issues the corresponding asset. 
+See [getBrandInfo](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#getBrandInfo).
 
 ```javascript
-const stakeAndSwapFn = async (orch, ...) => {
-...
-  const omni = await orch.getChain('omniflixhub');
-  const agoric = await orch.getChain('agoric');
+const brandInfo = orchestrator.getBrandInfo('denom');
+```
 
-  const [omniAccount, localAccount] = await Promise.all([
-    omni.makeAccount(),
-    agoric.makeAccount(),
-  ]);
+- `asAmount`converts a denom amount to an `Amount` with a brand. See [asAmount](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#asAmount).
 
-  const omniAddress = omniAccount.getAddress();
+```javascript
+const amount = orchestrator.asAmount({ denom: 'uatom', value: 1000n });
+```
 
-  // deposit funds from user seat to LocalChainAccount
-...
-  const transferMsg = orcUtils.makeOsmosisSwap({ ... });
+## Orchestration Account
 
-  try {
-    await localAccount.transferSteps(give.Stable, transferMsg);
-    await omniAccount.delegate(offerArgs.validator, offerArgs.staked);
-  } catch (e) {
-    console.error(e);
-  }
-};
+Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the 
+[`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI) interface. These accounts provide high-level operations for managing 
+accounts on remote chains, allowing seamless interaction and management of interchain
+accounts. The orchestration accounts abstract the complexity of interchain interactions,
+providing a unified and simplified interface for developers.
+
+### 1. Address Management
```

Comment:
To be consistent with the no-numbering in the previous section, the Orchestrator Interface
```suggestion
### Address Management
```